src/main/java/org/fundacion/jala/converter/models/AssetSQL.java

```diff
@@ -76,9 +80,9 @@ public static void deleteProject(final int assetId) {
     }
 
     /**
-     * Lists the project in the db.
+     * Lists all the date of the audio and video converter in the db.
      *
-     * @return a list of projects.
+     * @return a list of the all the date of the database.
```

Comment:
It would be better to put:
a list with all the data of the database. 
Or
a list of all the data from the database.

Reply:
it was solved